lib/utils/telemetry/generatePayload.js

```diff
@@ -300,17 +300,19 @@ module.exports = ({
     payload.isConfigValid = getConfigurationValidationResult(configuration);
     payload.dashboard.orgUid = serverless && serverless.service.orgUid;
 
-    if (serverless && command === 'deploy' && isAwsProvider) {
+    if (isAwsProvider && serverless && (command === 'deploy' || command === '')) {
       const serviceName = isObject(configuration.service)
         ? configuration.service.name
         : configuration.service;
-      const accountId = serverless.getProvider('aws').accountId;
+      const accountId = serverless && serverless.getProvider('aws').accountId;
       if (serviceName && accountId) {
         payload.projectId = crypto
           .createHash('sha256')
           .update(`${serviceName}-${accountId}`)
           .digest('base64');
       }
+
+      payload.didCreateStack = Boolean(serverless && serverless.getProvider('aws').didCreateStack);
```

Comment:
In platform PR I've suggested that maybe it'll be better to give it a more agnostic name as `didCreateService`, so if eventually later we will support non-CF-related deployments, this name stays accurate.

WDYT?

Reply:
Not sure if that applies to the metric name as well, but for the metric name +1 to have it a bit more generic

Reply:
Great suggestion @medikoo 👍